### PR TITLE
Added grunt bump

### PR DESCRIPTION
**Pull request by marianeaj**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -20,6 +21,26 @@ module.exports = function(grunt) {
             coverage: ['test/coverage.html']
         },
 
+        //Bump up version of npm module
+        bump: {
+            options: {
+                files: ['package.json'],
```
Comment (djvirgen) [inline Gruntfile.js]: Also need to bump bower.json, the version numbers should always be in sync.

File: Gruntfile.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ module.exports = function(grunt) {
     grunt.loadNpmTasks('grunt-docco');
     grunt.loadNpmTasks('grunt-mocha-test');
     grunt.loadNpmTasks('grunt-webpack');
+    grunt.loadNpmTasks('grunt-bump');
```
Comment (djvirgen) [inline Gruntfile.js]: Any package.json changes for this module?

Comment (marianeaj) [inline Gruntfile.js]: grunt bump is already installed in package.json... i found that weird too

File: bower.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "monocle-client-js",
-  "version": "0.1.3",
+  "version": "0.1.4",
```
Comment (marianeaj) [inline bower.json]: updating this version cause i already bumbed monocle-client-js previously.... do i need to do anything else?

**Comment by djvirgen** [comment]
Looks good! 🍒

**Pull request merged**